app/assets/javascripts/hyrax/save_work/save_work_control.es6

```diff
@@ -92,6 +92,23 @@ export default class SaveWorkControl {
     this.preventSubmit()
     this.watchMultivaluedFields()
     this.formChanged()
+    this.refererAnchor = this.addRefererAnchor()
```

Comment:
There's a lot of repeated code across the es6 files that have been changed in this branch. Have you considered creating a new es6 class with the common code and using it in each of these other es6 files?